Gemfile

```diff
@@ -114,7 +114,7 @@ group :development do
   gem "brakeman", "~> 4.10", require: false # Brakeman detects security vulnerabilities in Ruby on Rails applications via static analysis
   gem "bundler-audit", "~> 0.7" # bundler-audit provides patch-level verification for Bundled apps
   gem "derailed_benchmarks", "~> 1.8", require: false # A series of things you can use to benchmark a Rails or Ruby app
-  gem "erb_lint", "~> 0.0.34", require: false # ERB Linter tool
+  gem "erb_lint", github: "thepracticaldev/erb-lint", branch: "jess/remove-version-pinning", require: false # ERB Linter tool
```

Comment:
Question: Is this the fix for erb lint? Also, shouldn't `thepracticaldev/erb-lint` be `forem/erb-lint`? Also, I'm not following why it's a branch of Jess'

Reply:
@nickytonline this is in there because `erb_lint` hasn't been updated. There's a working version that was submit as a PR, @maestromac correctly suggested to fork the PR to avoid including temporary code we don't have control on. It was forked over `thepracticaldev` because I don't have the rights to fork on `forem/` and this will be corrected soon, as soon as `erb_lint` releases a new version

Reply:
Basically a magician trick :D